### PR TITLE
Gh 313

### PR DESCRIPTION
**Pull request by brenuart**
Changes applied. Please review.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -324,4 +324,59 @@
...
+		Activation:
+			The M2E Eclipse plugin exports the property "m2e.version" automatically when installed.
+			This profile is automatically activated when that property is found, meaning the build
+			is running under Eclipse...
+		
+		Additional doc:
+			https://docs.sonatype.org/display/M2ECLIPSE/Customizable+build+lifecycle+mapping+for+m2e+extensions+developers
+			http://wiki.eclipse.org/M2E
+	 -->
+	<profiles>
```
Comment (spencergibb) [inline pom.xml]: Can you do a separate pull request for this?  I'm not sure this project is the right place anyway.  [spring-cloud-build](https://github.com/spring-cloud/spring-cloud-build) might be more appropriate.

File: spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/RibbonClientConfiguration.java [hunk 2]
```diff
@@ -88,6 +89,7 @@ public IPing ribbonPing(IClientConfig config) {
 	}
 
 	@Bean
+	@Lazy // see https://github.com/spring-cloud/spring-cloud-netflix/issues/312
```
Comment (spencergibb) [inline spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/RibbonClientConfiguration.java]: I'd rather a short explanation than a url to an issue

Comment (brenuart) [inline spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/RibbonClientConfiguration.java]: See PR #316

File: pom.xml (hunk 1 above)
Comment (brenuart) [inline pom.xml]: You are right. New PR created against `spring-cloud-build`

**Comment by brenuart** [comment]
Closing this pull request since all changes are now covered by other PR (sorry for the confusion, still learning how Github works ;-)
